### PR TITLE
Initialized gems and file structure

### PR DESCRIPTION
**Pull request by IshmeetSingh06**
This branch contains the initial gem file setup

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 *.gem
 *.rbc
+*.lock
```
Comment (gagan93jtg) [inline .gitignore]: Gemfile.lock is [necessary to be committed](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore)

Comment (IshmeetSingh06) [reply .gitignore]: I have removed this from .gitignore

**Review by gagan93jtg — APPROVED** [review]
LGTM. Let's wait for Praman to approve.

File: Gemfile [hunk 2]
```diff
@@ -2,4 +2,7 @@
 
 source "https://rubygems.org"
 
-# gem "rails"
+gem "pg"
+gem "rspec"
+gem "simplecov"
+gem "bcrypt"
```
Comment (PramanSatya) [inline Gemfile:8]: add extra line at end of file to avoid this red symbol.
We leave one extra line at end of file as a convention

**Pull request merged**